Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 ## [Unreleased]
+  - Added the following real-time response API commands:
```

Comment:
we dont use changelog anymore, need to move this to the release notes
see https://xsoar.pan.dev/docs/integrations/changelog